test/acceptance/retries.js

```diff
@@ -0,0 +1,62 @@
+describe('retries', function() {
```

Comment:
Could this be moved to test/integration as a test fixture? Makes it easier to verify test output